src/ruby/ruby.i

```diff
@@ -1,4 +1,4 @@
-%module "ignition::math"
+%module "gz::math"
```

Comment:
I don't know if this has packaging consequences, we may need to update the `-release` repo after this is merged